tests/_helpers/assertions.zsh

```diff
@@ -3,7 +3,9 @@ _zunit_assert_mock_times() {
     local count=$2
     shift 2
 
-    local len=$(cat -- ${target}_mock_times)
+    : ${mock_dir:=${funcsourcetrace[1]:P:h:h}/_support/mock}
```

Comment:
It's defined at the top of mock.zsh. Why is it there?

Reply:
I think about it, they are unnecessary.